raw/GB/language-tables/Moksha.csv

```diff
@@ -1,196 +1,196 @@
-GB262,Is there a clause-initial polar interrogative particle?,0,Bartens 1999:176,,,,,
-GB263,Is there a clause-final polar interrogative particle?,0,Bartens 1999:176,,,,,
-GB264,Is there a polar interrogative particle that most commonly occurs neither clause-initially nor clause-finally?,0,Bartens 1999:176,,,,,
-GB265,"Is there a comparative construction that includes a form that elsewhere means 'surpass, exceed'?",0,Bartens 1999:176,,,,,
-GB266,Is there a comparative construction that employs a marker of the standard which elsewhere has a locational meaning?,1,Bartens 1999:109,,GB276,,,
-GB270,Can comparatives be expressed using two conjoined clauses?,0,Bartens 1999:109,,,,,
-GB273,Is there a comparative construction with a standard marker that elsewhere has neither a locational meaning nor a 'surpass/exceed' meaning?,0,Bartens 1999:109,,,,,
-GB275,Is there a bound comparative degree marker on the property word in a comparative construction?,0,Bartens 1999:109,,,,,
-GB276,Is there a non-bound comparative degree marker modifying the property word in a comparative construction?,1,Bartens 1999:109,"The demonstrative śä in the ablative case, preceding the adjective.",GB276,,,
-GB285,Can polar interrogation be marked by a question particle and verbal morphology?,0,Bartens 1999:176,,,,,
-GB286,Can polar interrogation be indicated by overt verbal morphology only?,0,Bartens 1999:176,,,,,
-GB291,Can polar interrogation be marked by tone?,0,Bartens 1999:176,,,,,
-GB296,Is there a phonologically or morphosyntactically definable class of ideophones that includes ideophones depicting imagery beyond sound?,0,Bartens 1999:168,,,,,
-GB297,Can polar interrogation be indicated by a V-not-V construction?,0,Bartens 1999:176,,,,,
-GB298,"Can standard negation be marked by an inflecting word (""auxiliary verb"")?",1,"Bartens 1999:140, 142","Among the tenses in preterite and among the moods in conjunctive, desiderative, imperative and optative.","GB298-1, GB298-2",,,
-GB299,"Can standard negation be marked by a non-inflecting word (""auxiliary particle"")?",1,Bartens 1999:141-142,"Among the tenses in present, 2nd preterite, among the moods in conditional and conditional-conjunctive.","GB299-1, GB299-2",,,
-GB300,Does the verb for 'give' have suppletive verb forms?,0,Bartens 1999:in passim,,,,,
-GB301,Is there an inclusory construction?,0,Bartens 1999:in passim,,,,,
-GB302,Is there a phonologically free passive marker ('particle' or 'auxiliary')?,0,Bartens 1999:158-162,,,,,
-GB303,Is there a phonologically free antipassive marker ('particle' or 'auxiliary')?,0,Bartens 1999:in passim,,,,,
-GB304,Can the agent be expressed overtly in a passive clause?,1,Toldova et al. 2018:490,,GB304,,,
-GB305,Is there a phonologically independent reflexive pronoun?,1,Bartens 1999:114,,GB305,,,
-GB306,Is there a phonologically independent non-bipartite reciprocal pronoun?, two + two)?,0,Bartens 1999:114,,,,

```

Comment:
Again, looks like the title is simply cut at the first comma, rather than quoting it and keeping the inner comma.